CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
+# Changelog
+
 ## [Unreleased]
 
+## [0.4.0] - 2023-08-04
+
```

Comment:
It seems that we will open another PR to close the version...
Then, i suggest to let this in above unreleased section 😉